src/components/Toolbar/index.js

```diff
@@ -98,6 +98,20 @@ export default class Toolbar extends Component {
     );
   }
 
+  renderKeybindMenu() {
+    const {vimMode, vimInsert} = this.props;
+    return (
+      <ActionMenu
+        label={vimMode ? `VIM ${vimInsert ? 'Insert' : 'Normal'}` : 'Normal'}
```

Comment:
I think VIM Normal is a bit confusing at first glance. maybe strings can be
- `Normal`
- `VIM`
- `VIM (insert)`

Reply:
Fixed!